cfg_mgmt/btp_application_certificate.py

```diff
@@ -55,16 +55,22 @@ def _setup_oauth_token(self):
         self.access_token = resp.json()['access_token']
 
     def create_client_certificate_chain(self, csr_pem: str, validity_in_days: int) -> dict:
+        '''
+        see https://pages.github.tools.sap/sap-pki-certificate-service/consumer-guide/getting-started/get-client-certificate/
```

Comment:
Please refrain from linking non-public resources.
Rather link this as comment within configuration.

Reply:
Why is this project on a public repo?